backend/app/anti_recommenders/arkg/arkg_anti_recommender.py

```diff
@@ -46,19 +47,20 @@ def __load_store(*, file_path: Path, mime_type: RdfMimeType) -> ox.Store:
         return store
 
     def __retrieve_anti_recommendations_from_store(
-        self, record_key: RecordKey
+        self, *, record_key: RecordKey, language: LanguageAlpha2
     ) -> tuple[RecordKey, ...]:
         """Return a tuple of anti-recommendations that have been retrieved from an ARKG Store."""
 
         return tuple(
-            binding["name"].value
+            str(binding["name"].value).replace("@" + language, "")
```

Comment:
Something's amiss if you have to do this. In RDF a literal is actually a tuple (string value, language XOR datatype), where language is the two-character ISO code and datatype is an IRI such as xsd:boolean specifying the datatype of the value (if not xsd:string). You probably just need to dig further into the literal struct to get the string value.

Reply:
Yes, you're right. I didn't need to use `replace` at all.